packages/codegen/src/EntityDbMetadata.ts

```diff
@@ -170,6 +170,7 @@ export class EntityDbMetadata {
   polymorphics: PolymorphicField[];
   tableName: string;
   tagName: string;
+  disableTags: boolean;
```

Comment:
Instead of introducing `disableTags`, maybe make `tagName: string | undefined`, and we can treat `tagName: undefined` as "don't use a tag".